packages/main/src/Switch.js

```diff
@@ -120,6 +120,19 @@ const metadata = {
 			type: String,
 			defaultValue: "",
 		},
+
+		/**
+		 * Defines the tooltip of the component.
+		 * <br>
+		 * <b>Note:</b> If applicable an external label reference should always be the preferred option to provide context to the <code>ui5-switch</code> component over a tooltip.
+		 * @type {string}
+		 * @defaultvalue: ""
+		 * @public
+		 * @since 1.9.0
+		 */
```

Comment:
In the case we need to provide API for setting the tooltip, we often provide a private property, called "title".
And because it's called "title" we operate with the value of the standard HTML "title" attribute. This is done in several components already: see Option, TreeItem, ListItem, SideNavigationItem

/**
		 * Defines the tooltip of the component.
		 * @type {string}
		 * @defaultvalue ""
		 * @private
		 */
		title: {
			type: String,
		},
		
There is a  "tooltip" in Button, can't recall the reason, but at least initially you can support the native way of setting a tooltip - with the "title" via the private "title" property. In case there is some specific things that requires a separate property, you can always do that.

```html
<ui5-switch title="my cool tooltip"></ui5-switch>
```
Advantages
- native way for setting tooltip
- no public API
- tooltip property can be always added later if required

Reply:
On second thought, there might be duplicate reading with Jaws... so yes it may not work for you. If that's the case "tooltip" is fine.

Reply:
Checked it with JAWS and it's read out twice if we use title. I think we have to stick to the tooltip solution

Reply:
Yes, I approve +1